staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -538,7 +520,7 @@ See http://k8s.io/docs/reference/using-api/api-concepts/#conflicts`, err)
 				Cascade:       o.DeleteOptions.Cascade,
 				Timeout:       o.DeleteOptions.Timeout,
 				GracePeriod:   o.DeleteOptions.GracePeriod,
-				ServerDryRun:  o.ServerDryRun,
+				DryRunServer:  o.DryRunStrategy.Server(),
```

Comment:
Do you still need this variable since it's not embedded in the helper?

Reply:
The pruner doesn't use a Visitor, and also uses a DynamicClient instead of a Helper. Sean's refactor will help with this https://github.com/kubernetes/kubernetes/pull/86361